src/components/CountryDetail.vue

```diff
@@ -1,7 +1,17 @@
 <template>
-    <div v-if="data">
-      <p> In {{year}}, {{countryName}} had ... Amenia and ... protein data</p>
+    <div v-if="data">   
+    <b-jumbotron >
+      <h2> {{countryName}}, {{year}} </h2>
+      <p> Prevalence of anemia: {{anemia}}% </p>
+      <p> Protein intake: {{proteinIntake}} (g/cap/day) </p>
+      <p> GDP: {{gdp}} (Trumps$) </p>
+    </b-jumbotron>
     </div>
+    <div v-else>
+       <b-jumbotron >
+      <p> Pick the region , country and year i gonna show you something cool </p>
```

Comment:
> Pick a country to view information
oder sowas in der Art. Ein Jahr sollte ja (sobald wir das gefixt sein) eh immer ausgewählt sein...